projects/angular-geocoder-autocomplete/src/lib/geocoder-autocomplete.component.ts

```diff
@@ -254,6 +261,11 @@ export class GeocoderAutocompleteComponent implements OnInit, AfterViewInit, OnC
       this.autocomplete.setLimit(changes['limit'].currentValue);
     }
 
+    if (changes['debounceDelay'] &&
```

Comment:
The  "debounceDelay" is an initial option, so we do not need to watch changes on it.
It was by design that the "autocomplete" library doesn't have a method for that.
The "setLimit" method limits the number of returned results.

Just remove watching the changes and we will merge your brunch. Thank you for contributing.